cli/js/stat.ts

```diff
@@ -10,9 +10,18 @@ export interface StatResponse {
   modified: number;
   accessed: number;
   created: number;
-  mode: number;
-  hasMode: boolean; // false on windows
   name: string | null;
+  isUnix: boolean; // false on windows
```

Comment:
Why isUnix? Is that in Node?

Reply:
But it seems unnecessary - we have ways of checking if windows without including this: `Deno.build.os`

Reply:
Makes sense.